### ISSUE TITLE
Subscription to Solidity events is giving back wrong argument values

### ISSUE THREAD

**Issue by marc4gov** [monax/legacy-contracts.js#83]
# Library Issue or Support Request?

This is the place to report any issues with the eris-contracts.js JavaScript library.  If you need help with your Solidity code, please see Monax's [Premium Support & Education](https://monax.io/packages/) offering.

# Version Numbers

Fill in version numbers below.

* Node.js: 7.2.0
* Eris CLI: 0.12.0
* eris-contracts: 0.15.9

# Issue Description

Subscription to events is giving back wrong argument values in strings and addresses, block numbers and block timestamps. Seems like some bit shift. 

# Code

Include all code and data needed to reproduce the problem by replacing the examples below.

## Solidity

```Solidity
....
    event TaskStatusSet(address taskAddress, bytes32 taskTitle, uint8 status);
...
    function setStatus(address taskAddress, address confirmer, uint8 status) {
        Task t = Task(taskAddress);
        t.setStatus(status);
        TaskStatusSet(taskAddress, t.title(), status);
    }

```

## JavaScript

```
exports.subToTaskStatusSet = function(){

    Blockchain.taskmanagerContract.TaskStatusSet(startCallback, eventCallback);

    function startCallback(error, eventSub){}

    function eventCallback(error, event){
        var evt = new StatusEvent({
            eventName: event.event,
            entity: "Task",
            address: event.args.taskAddress,
            status: event.args.status,
            name: eriswrapper.hex2str(event.args.taskTitle),
        });
        saveToDb(evt);
    }
}
```

Results in console.log:

```
OPTIONS /tasks 200 0.399 ms - 17
{ title: 'taak1',
  address: '1C1824985F812FE5714B3C0FDB97025B14A164EA',
...
```

Result in Mongo DB:

```
{
    "_id" : ObjectId("58d00098994967b5adebb890"),
    "eventName" : "TaskStatusSet",
    "address" : "1824985F812FE5714B3C0FDB97025B14A164EA74",
    "time" : ISODate("2017-03-20T16:17:28.406Z"),
    "name" : "aak1",
    "confirmer" : "",
    "id" : 0,
    "status" : 1,
    "entity" : "Task",
    "__v" : 0
}

```

As you can see, the address is converted from `1C1824985F812FE5714B3C0FDB97025B14A164EA` to `1824985F812FE5714B3C0FDB97025B14A164EA74` which looks like some bit shift
and the title of the Task is 'taak1' which converts to 'aak1' using `eris-wrapper.hex2str` function

**Comment by NodeGuy** [comment]
I believe this has been fixed in 0.15.12.  Please try it and let me know.

**Comment by marc4gov** [comment]
Correct! It's fixed, thanks!
